eras/mary/impl/src/Cardano/Ledger/Mary/Value.hs

```diff
@@ -135,23 +135,36 @@ instance DecCBOR AssetName where
 
 -- | Policy ID
 newtype PolicyID c = PolicyID {policyID :: ScriptHash c}
-  deriving (Show, Eq, EncCBOR, DecCBOR, Ord, NoThunks, NFData, Generic)
+  deriving
+    ( Show
+    , Eq
+    , Ord
+    , Generic
+    , NoThunks
+    , NFData
+    , EncCBOR
+    , DecCBOR
+    , ToJSON
```

Comment:
Are the default implementations of `ToJSON` and `ToJSONKey`  the same as [the one we removed](https://github.com/input-output-hk/cardano-ledger/pull/3338/commits/d1ca75f073cb84b7eab70dd83b4522dc3df71b75#diff-abef920a14ec558ef5e82f62b48e09747f4683b5f60ae07e370fc2e6b70c2f64L401)? 
```
toJSON (PolicyID (ScriptHash h)) = Aeson.String (Hash.hashToTextAsHex h)

toJSONKey = toJSONKeyText render
    where
      render (PolicyID (ScriptHash h)) = Hash.hashToTextAsHex 
```

Reply:
Good question and yes they are. I confirmed it before switching to the derived method:
https://github.com/input-output-hk/cardano-base/blob/dd15fa63bafad152873f6f257b3fed6ef37ebe78/cardano-crypto-class/src/Cardano/Crypto/Hash/Class.hs#L304-L312